api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -780,12 +779,12 @@ def update_venue(venue_id: int) -> utils.BackofficeResponse:
             return render_venue_details(venue_row, form), 400
 
         try:
-            if not sirene.siret_is_active(new_siret):
+            if not entreprise_api.get_siret_open_data(new_siret).active:
```

Comment:
Ici on n'a a priori pas besoin de l'adresse : 
```suggestion
            if not entreprise_api.get_siret_open_data(new_siret, with_address=False).active:
```

Reply:
Ici on va cherche un SIRET et non un SIREN, on n'a pas de paramètre `with_address` pour ajouter le siège social